### ISSUE TITLE
Error - No such file or directory: 'git'

### ISSUE THREAD

**Issue by frosenbrock** [ChrisYounger/git_for_splunk#5]
I am a Splunk beginner and have difficulties with git_for_splunk. Following the instructions, but the application cannot find the .git folder

GIT_WORK_TREE = / opt / splunk / etc / apps / dk_app_mb /
GIT_DIR = / opt / splunk / etc / apps / dk_app_mb / .git /
An exception of type FileNotFoundError occurred. Arguments:
(2, "No such file or directory: 'git'")

I also tried:
GIT_DIR = / opt / splunk / etc / apps / dk_app_mb /
splunk container folder

![image](https://user-images.githubusercontent.com/39100550/92787830-d3ead480-f37f-11ea-8107-78a40e75cf76.png)

**Comment by ChrisYounger** [comment]
Hi frosenbrock,

Sorry looks like my instructions are not clear enough. Based on your screenshot, your GIT_WORK_TREE should be `etc/` and the GIT_DIR should be `.git/`

Let me know if this does not solve the problem
Chris

**Comment by frosenbrock** [comment]
sorry, I put the wrong image

My application folder and git directory:
splunk/etc/apps/dk_app_mb#
appserver
bin
btool_output
default
.git
.gitignore
local
lookups
metadata
static

And so is my configuration

![image](https://user-images.githubusercontent.com/39100550/94194620-54cdc400-fe88-11ea-9e76-3a25c03e0bdf.png)

and the log

GitForSplunk 1600977988
SPLUNK_HOME=/opt/splunk
GIT_WORK_TREE=/opt/splunk/etc/apps/dk_app_mb/
GIT_DIR=/opt/splunk/etc/apps/dk_app_mb/.git/
An exception of type FileNotFoundError occurred. Arguments:
(2, "No such file or directory: 'git'")
runtime=0.0024430000000000285
status=2

**Comment by ChrisYounger** [comment]
ok it looks like Splunk cant find the git executable. Are you running this inside a docker container that doesn't have git installed?

**Comment by frosenbrock** [comment]
Yes, that is the problem 🤨😁 . With the installation of the package on windows, was successful. I haven't been able to install Git on the Docker Container yet. I will make updates on this topic about my procedure.
